mlir/include/mlir/Conversion/Passes.td

```diff
@@ -139,24 +139,7 @@ def ArithToAMDGPUConversionPass : Pass<"convert-arith-to-amdgpu"> {
 
 def ConvertArithToEmitC : Pass<"convert-arith-to-emitc"> {
   let summary = "Convert Arith dialect to EmitC dialect";
-  let description = [{
-    This pass converts `arith` dialect operations to `emitc`.
```

Comment:
We should keep a short description

Reply:
I restored it but isn't it redundant with the summary? I don't know where this is actually used (doxygen maybe?).

Reply:
Not sure if there is a policy. I was under the impression that all passes have a description...?

Reply:
I honestly don't know either. That one currently doesn't have a description upstream.

Reply:
I think this is a remainder of the PR I raised upstream to introduce ArithToEmitC, but then the other versioning was merged, so only we have the description.

Reply:
Sounds good to go with what upstream has